native/src/network.c

```diff
@@ -79,6 +80,15 @@ static apr_status_t sp_socket_cleanup(void *data)
         (*s->net->cleanup)(s->opaque);
     if (s->sock) {
         apr_socket_t *as = s->sock;
+        apr_sockaddr_t *sa = NULL;
+        apr_socket_addr_get(&sa, APR_LOCAL, as);
+        if (sa && sa->family == APR_UNIX) {
```

Comment:
If you guard all the rest with `#ifdef APR_UNIX` why not this one?

Reply:
This one is a mistake.